pkg/option/config.go

```diff
@@ -1300,6 +1300,10 @@ func (c *DaemonConfig) Validate() error {
 		return fmt.Errorf("MTU '%d' cannot be negative", c.MTU)
 	}
 
+	if c.IPAM == IPAMENI && c.EnableIPv6 {
```

Comment:
Is this a limitation on the AWS side? A comment linking to according documentation on this restriction would be helpful.

Reply:
AWS does support IPv6 but it requires manual steps. However, Cilium currently doesn't support it. I'll add a note to the ENI guide.

Reply:
Documented.